aries-site/src/examples/cardPreviews/textinput.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { TextInput, FormField } from 'grommet';
+
+export const TextInputPreview = () => {
+  return (
+    <FormField
+      style={{ borderRadius: '4px' }}
+      fill="horizontal"
+      border={{ color: 'focus' }}
```

Comment:
Any way we can make this border a little thicker? it feels more narrow than a typical focus outline